pombola/south_africa/templates/core/organisation_detail.html

```diff
@@ -39,6 +39,22 @@
     {% include "core/_position_listing.html" with positions=positions %}
   {% endif %}
 
+  <div class="pagination">
+    <span class="step-links">
+      {% if positions.has_previous %}
+        <a href="?page={{ positions.previous_page_number }}">previous</a>
+      {% endif %}
+
+      <span class="current">
```

Comment:
I've taken out the step-links span and I think it looks fine (I quite like the button look - the mouse pointer was in the "previous" link which is why it's highlighted):

![pagination-2](https://cloud.githubusercontent.com/assets/7907/6216770/dd0e4fd8-b604-11e4-9a43-c06773b84c85.png)

Do you think that'll do? Otherwise, not having the button styling, but em-dashes after previous or before next (removing the full stop) looked fine too